### PR TITLE
fix(discover) Format y-axis and tooltips consistently

### PR DESCRIPTION
**Pull request by markstory**
Update discover and performance views to format durations as s/ms consistently across all charts. Also use short forms for time units in tooltips and axis labels as they consume less space and are more consistent with how we show these values elsewhere.

#### failure rate as %
Screen Shot 2020-07-30 at 9 22 20 PM

#### discover with durations
Screen Shot 2020-07-30 at 5 53 09 PM

#### Performance overview
Screen Shot 2020-07-30 at 5 34 44 PM

#### Performance summary
Screen Shot 2020-07-30 at 5 22 23 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/utils/discover/charts.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import {getDuration, formatPercentage} from 'app/utils/formatters';
+import {aggregateOutputType} from 'app/utils/discover/fields';
+
+/**
+ * Formatter for chart tooltips that handle a variety of discover result values
+ */
+export function tooltipFormatter(value: number, seriesName: string): string {
+  switch (aggregateOutputType(seriesName)) {
+    case 'number':
```
Comment (Zylphrex) [inline src/sentry/static/sentry/app/utils/discover/charts.tsx:9]: Did you miss a `case 'integer':` here or was that intended?

File: tests/js/spec/utils/formatters.spec.jsx [hunk 2]
```diff
@@ -2,8 +2,39 @@ import {
...
   formatAbbreviatedNumber,
   formatPercentage,
+  getDuration,
 } from 'app/utils/formatters';
 
+describe('getDuration()', function() {
+  it('should format durations', function() {
+    expect(getDuration(0, 2)).toBe('0.00ms');
+    expect(getDuration(0.1)).toBe('100ms');
+    expect(getDuration(0.1, 2)).toBe('100.00ms');
```
Comment (Zylphrex) [inline tests/js/spec/utils/formatters.spec.jsx:12]: Do we want milliseconds to get special treatment here and use the abbreviated version?

**Review by Zylphrex — APPROVED** [review]

File: src/sentry/static/sentry/app/utils/discover/charts.tsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/utils/discover/charts.tsx:9]: It was missing from the code this was drawn from. We don't have any aggregates that create integer types right now, but it is good to have for completeness.

File: tests/js/spec/utils/formatters.spec.jsx (hunk 2 above)
Comment (markstory) [reply tests/js/spec/utils/formatters.spec.jsx:12]: This scenario was for when you ask for 2 decimal places. I didn't want to change the behavior of the formatter and was looking to better capture how it currently works as these functions had no tests before.

I'm undecided about how to handle time > 0ms and <1s for axis labels. Right now we output `300ms` which is quite a bit larger than 0 and 1s.

**Pull request merged**